### DISCUSSION TITLE
关于第四章主线剧情的感想

### DISCUSSION THREAD

**Discussion by RWHwaxili** [share121/inter-knot#5406] [绝区零]
想必各位已经把第四章主线剧情肝完了吧，想问一下各位，在完成外环剧情回家时去旧都废墟.大裂谷的那段剧情，大家作何感想

**Comment by aino2333** [comment]
主线挺好的，唯一的遗憾就是策划说要把主线走格子全砍掉😭我还是希望主线至少能有一点走格子

**Comment by March-777th** [comment]
也是图穷匕见了

**Comment by cpu644** [comment]
填个坑 我蛮看好的 就是感觉第四章比1/2/3哪个都短 一个小时就过完了
